styles/admin.css

```diff
@@ -0,0 +1,52 @@
+.nav {
+    background-color: #17243A;
+    height: 5em;
+}
+
+#name {
+    text-align: right;
+    padding-right: 2em;
+    color: white;
+}
+
+#account-info.col-xs-6 {
+    padding-right: 1em;
+}
+
+#logo {
+    padding-left: 2em;
+    color: white;
+}
+
+#logo:hover {
+    cursor: pointer;
+}
+
+#image-dir {
+    border: 1px solid black;
+    height: 10em;
+}
+
+#user-list {
+    border: 1px solid black;
+    height: 10em;
+}
+
+#assign-batch {
+    margin: auto;
+    width: 60%;
+    font-size: 2rem;
+    text-align: center;
+    font-family: "Open Sans", sans-serif;
+    border-radius: 1em;
+    line-height: 3rem;
+    transition: all 0.3s cubic-bezier(.25,.8,.25,1);
```

Comment:
Open Sans is part of the [Google Fonts pack](https://fonts.google.com/). You add it in with an HTML tag in the header.